build_tools/docker/base/Dockerfile

```diff
@@ -18,37 +18,37 @@ ENV CXX /usr/bin/clang++
 
 RUN apt-get update \
   && apt-get install -y \
-    # For updating IREE's submodules.
-    git \
-    # Default compiler and linker.
-    clang \
-    lld-9 \
-    # IREE transitive dependencies
-    libsdl2-dev \
-    libssl-dev \
-    # A much better CMake builder
-    ninja-build \
-    # For building child images. Best practices would tell us to use multi-stage
-    # builds (https://docs.docker.com/develop/develop-images/multistage-build/)
-    # but it turns out that Dockerfile is a thoroughly non-composable awful
-    # format and that doesn't actually work that well. These deps are pretty
-    # small.
-    unzip \
-    wget \
-    gnupg2 \
-    # Needed for installing Bazel, per https://bazel.build/install/ubuntu
-    apt-transport-https \
-    curl \
-    gnupg \
-    # Needed for building lld with Bazel (as currently configured)
-    libxml2-dev \
-    # Optional for tools like llvm-symbolizer, which we could build from
-    # source but would rather just have available ahead of time
-    llvm-dev \
-    # Someone is welcome to tell me a better way to just install lld-9 as lld
-    # (lld=9 doesn't work)
-    && ln -s lld-9 /usr/bin/lld \
-    && ln -s ld.lld-9 /usr/bin/ld.lld
+  # For updating IREE's submodules.
```

Comment:
Yeah there isn't a canonical format for Dockerfile AFAIK and these indents were intentional. I disabled formatting for docker files because it was doing some weird things. If we want to standardize on some formatter that would be OK though

Reply:
Okay, I reverted the whitespace changes and will disable whatever formatting my IDE was performing.